osdu/client/aws.py

```diff
@@ -58,7 +60,7 @@ def get_tokens(self, password) -> None:
         response = cognito.initiate_auth(
```

Comment:
In order to make `secret_hash` optional:

```suggestion
        auth_params = { 
            'USERNAME': self._user, 
            'PASSWORD': password
        }
        if secret_hash:
            auth_params['SECRET_HASH'] = secret_hash
        
        response = cognito.initiate_auth(
            AuthFlow='USER_PASSWORD_AUTH',
            ClientId=self._client_id,
            AuthParameters=auth_params
        )
```